### ISSUE TITLE
Unable to control GPU fans anymore - This control does not respond properly to the current command

### ISSUE THREAD

**Issue by 24kris6** [Rem0o/FanControl.Releases#2797]
**Describe the bug**
Since recently, I'm not able to use FanControl to control my GPU fans. I'm not completely sure, but it's possible that the issue started happening after Windows 11 24H2 update. I know for a fact it used to work without problems, I'm just not sure what changed.
If it matters, all my case fans still work correctly with FanControl, it's just the GPU fans that are problematic.

"Force apply" doesn't appear to make a difference. 

The GPU is a KFA2 RTX 4070 Ti Super.

**Comment by Woffingshire** [comment]
I am getting the exact same issue with an RX 6800 XT

**Comment by Rem0o** [comment]
"How" doesn't it work? If you crank it all the way to a 100%, it doesn't do anything?

**Comment by systemerror-codenotfound** [comment]
@Rem0o 
Sure, let me detail it in my case as i was making some more tests
Both fans are connected directly to the GPU
![image](https://github.com/user-attachments/assets/31e8c022-52fa-4cce-aefe-9e223d513df2)

They can't get below 55% in my case, i don't know exactly why and they were able to do it
I have 3 noctua 80mm fans (1 connected to 1 connector and in the other one i use a splitter to connect 2 fans), they can go as 30% low and they will stop

If i do a calibration test it works as intented, the fan goes 100% PWM 2000RPM down to 30% 0RPM
![image](https://github.com/user-attachments/assets/30276f82-5e50-474a-bbd2-c2ffdc9526bd)

But when i try to set in manually or using a curve, it doesn't respond below 37% and it automatically default to 55%
![image](https://github.com/user-attachments/assets/9390386f-dc46-4bf1-bde0-c6cde4260224)

Using other Fan control software i can control it without any problem, so it's not related with the GPU

**Comment by Rem0o** [comment]
Expand the control cards and see what you got as start and stop %. Try and reset these parameters by putting 0 in. Also make sure you got the 0% override checkbox on in sensor source in the settings page.

**Comment by systemerror-codenotfound** [comment]
@Rem0o  changing the start % makes it workm, but as soon as i run a new fan calibration it broke it again, as start % is set to 42%

Now it defaults to 63%...
![image](https://github.com/user-attachments/assets/63cf83fe-fc4d-4a26-81b5-4199d7fbea5a)

**Comment by Rem0o** [comment]
The automatic calibration is not perfect.  It sweeps the percentage down until the fan stops from 70, then go back up until it starts again.

Now you may do the exact same thing manually with start and stop set to 0 and see what's going on.

**Comment by Woffingshire** [comment]
On AMD it just seems to not be playing nice with Adrenaline. About half the time it wouldn't apply the fan curve so the fans would act as if they had been set to the lowest possible RPM cause it to overheat as soon as I went on a game. The other half it would just apply the default Adrenaline fan curve for the card and ignore Fan Control.
Recalibrating the GPU didn't fix the issue. In fact a few times it said that it couldn't even find the GPU.

I fixed it by installing the deprecated ADLX plugin and recalibrating by GPU with the plugin active. It has worked perfectly since.
